examples/Acrobot/test/AcrobotWParams.urdf

```diff
@@ -6,6 +6,7 @@
 
   <parameter name="b1" value=".1" lb="0" />
   <parameter name="b2" value=".1" lb="0" />
+  <parameter name="l1" value="1" lb="0" />
   <parameter name="lc1" value=".5" lb="0" />
```

Comment:
Would it make sense to have the center of mass distance scale with the length of the arm?  We could zap lc1 and just have the CoM distance be computed as l1/2.